splashsurf_lib/src/uniform_grid.rs

```diff
@@ -115,6 +115,17 @@ pub struct UniformCartesianCubeGrid3d<I: Index, R: Real> {
     n_cells_per_dim: [I; 3],
 }
 
+impl<I: Index, R: Real> Default for UniformCartesianCubeGrid3d<I, R> {
+    fn default() -> Self {
+        Self {
+            aabb: AxisAlignedBoundingBox3d::new(Vector3::zeros(), Vector3::zeros()),
+            cell_size: R::zero(),
+            n_points_per_dim: [I::zero(); 3],
+            n_cells_per_dim: [I::zero(); 3],
+        }
+    }
+}
+
```

Comment:
I think it would replace this with some `pub(crate)` method, something like `Grid3d::new_zero()` or `Grid3d::new_empty()` to not expose this method to the user. Or do you have an idea for a different name?